app/src/main/java/com/android/streetworkapp/ui/map/MapUi.kt

```diff
@@ -290,54 +292,71 @@ fun ParkFilterSettings(userFilterInput: FilterSettings) {
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun MapSearchBar(query: MutableState<String>, onCancel: () -> Unit) {
-  Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.Center) {
-    SearchBar(
-        query = query.value,
-        onQueryChange = { query.value = it },
-        placeholder = { "Search for parks" },
-        modifier = Modifier.testTag("searchBar").padding(bottom = 8.dp).padding(horizontal = 4.dp),
-        active = false,
-        onActiveChange = {},
-        onSearch = {},
-        leadingIcon = {
-          IconButton(onClick = onCancel, modifier = Modifier.testTag("cancelSearchButton")) {
-            Icon(
-                imageVector = Icons.AutoMirrored.Filled.ArrowBack,
-                contentDescription = "Search",
-            )
-          }
-        },
-        colors = SearchBarDefaults.colors(containerColor = ColorPalette.INTERACTION_COLOR_LIGHT)) {}
-  }
+fun MapSearchBar(query: MutableState<String>) {
+  Row(
+      modifier = Modifier.fillMaxWidth().padding(vertical = 20.dp),
+      horizontalArrangement = Arrangement.Center) {
+        SearchBar(
+            query = query.value,
+            onQueryChange = { query.value = it },
+            placeholder = { Text("Search for parks") },
```

Comment:
Improvement: Since it's asked by the coaches, could you put this text in the strings.xml

I would do it like that:
```kotlin
Text(stringResource(R.string.park_search))
```
```xml
<string name="park_search">Search for parks</string>
```